src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,111 @@
+@use '../utils/variables' as *;
+@use '../utils/mixins' as *;
+
+.card {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  align-items: center;
+
+  width: $card-width;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  padding: 16px;
+
+  font-family: Roboto, sans-serif;
+
+  @include transition;
+
+  &__picture {
+    width: 160px;
+    height: 134px;
+
+    margin-top: $margin-top;
+  }
+
+  &__item-model {
+    font-weight: $medium-font-weight;
+    font-size: 12px;
+    line-height: $bigger-line-height;
+    color: rgba(6, 11, 53, 1);
+
+    margin-top: 40px;
+  }
+
+  &__item-code {
+    font-weight: $smaller-font-weight;
+    font-size: 10px;
+    line-height: $smaller-line-height;
+    color: rgba(97, 96, 112, 1);
+
+    width: $paragraph-width;
```

Comment:
Ensure that the `$paragraph-width` variable is defined in the `variables` file. If it's not defined, the width might not be applied correctly.